decidim-admin/app/views/decidim/admin/share_tokens/index.html.erb

```diff
@@ -0,0 +1,47 @@
+<div class="card share_tokens">
+  <div class="item_show__header">
+    <h2 class="item_show__header-title">
+      <%= t(".title", name: resource_title) %>
+      <a class="button button__sm button__secondary" href="<%= share_tokens_path(:new) %>"><%= icon "add-line" %><%= t(".new_share_token_button") %></a>
+    </h2>
+  </div>
+  <div class="mb-4">
+    <p class="help-text whitespace-pre-line"><%= t(".share_tokens_help_html", clipboard: icon("clipboard-line", class: "inline-block mb-1")) %></p>
+  </div>
+  <% if @share_tokens.any? %>
+    <div class="table-scroll">
+      <table class="table-list">
+        <thead>
+          <tr>
+            <th><%= sort_link(query, :token, t("models.share_token.fields.token", scope: "decidim.admin"), default_order: :desc) %></th>
+            <th class="w-20"><%= sort_link(query, :expires_at, t("models.share_token.fields.expires_at", scope: "decidim.admin"), default_order: :desc) %></th>
+            <th class="w-28"><%= sort_link(query, :registered_only, t("models.share_token.fields.registered_only", scope: "decidim.admin"), default_order: :desc) %></th>
+            <th class="w-20"><%= sort_link(query, :times_used, t("models.share_token.fields.times_used", scope: "decidim.admin"), default_order: :desc) %></th>
+            <th class="w-20"><%= t("models.share_token.fields.actions", scope: "decidim.admin") %></th>
```

Comment:
```suggestion
            <th class="w-25"><%= t("models.share_token.fields.actions", scope: "decidim.admin") %></th>
```

Reply:
I'll use w-32 everywhere so we leave a bit of margin

Reply:
i am not sure there are dimensions like w32/ w28 ... in my case i could see any effect when applying w20 , w25 , w30

Reply:
I always look at the tailwind spec for this:
![imatge](https://github.com/user-attachments/assets/fe002324-8f10-45aa-9d34-eb78869d2e47)

I've used w32 to give some margin to non-english languages (English is usually shorter)